### PR TITLE
[Mellanox] Add a new Mellanox platform x86_64-mlnx_msn4600c and new SKU ACS-MSN4600C

### PR DESCRIPTION
**Pull request by shlomibitton**
- What I did
Add a new Mellanox platform x86_64-mlnx_msn4600c and new SKU ACS-MSN4600C.

- How I did it

1.  Add related files to the device folder:

buffer config templates
pg lookup profile
port_config.ini
sai profile
sensor conf
plugins

2.  Edited related platform api files:

chassis.py
psu.py
thermal.py

- How to verify it

- Description for the changelog

- A picture of a cute animal (not mandatory but encouraged)

### CODE REVIEW

**Comment by shlomibitton** [comment]
@liat-grozovik @nazariig @keboliu 
please review

File: device/mellanox/x86_64-mlnx_msn2700-r0/plugins/sfputil.py [hunk 1]
```diff
@@ -41,7 +41,7 @@
 
 # magic code defnition for port number, qsfp port position of each hwsku
 # port_position_tuple = (PORT_START, QSFP_PORT_START, PORT_END, PORT_IN_BLOCK, EEPROM_OFFSET)
-hwsku_dict = {'ACS-MSN2700': 0, 'Mellanox-SN2700': 0, 'Mellanox-SN2700-D48C8': 0, "LS-SN2700":0, 'ACS-MSN2740': 0, 'ACS-MSN2100': 1, 'ACS-MSN2410': 2, 'ACS-MSN2010': 3, 'ACS-MSN3700': 0, 'ACS-MSN3700C': 0, 'ACS-MSN3800': 4, 'Mellanox-SN3800-D112C8': 4, 'ACS-MSN4700': 0, 'ACS-MSN3420': 5}
+hwsku_dict = {'ACS-MSN2700': 0, 'Mellanox-SN2700': 0, 'Mellanox-SN2700-D48C8': 0, "LS-SN2700":0, 'ACS-MSN2740': 0, 'ACS-MSN2100': 1, 'ACS-MSN2410': 2, 'ACS-MSN2010': 3, 'ACS-MSN3700': 0, 'ACS-MSN3700C': 0, 'ACS-MSN3800': 4, 'Mellanox-SN3800-D112C8': 4, 'ACS-MSN4700': 0, 'ACS-MSN3420': 5, 'ACS-MSN4600C': 4}
```
Comment (nazariig) [inline device/mellanox/x86_64-mlnx_msn2700-r0/plugins/sfputil.py:44]: @shlomibitton please replace double quotes with single for LS-SN2700

Comment (shlomibitton) [reply device/mellanox/x86_64-mlnx_msn2700-r0/plugins/sfputil.py]: fixed

**Comment by liat-grozovik** [comment]
retest vsimage please

**Pull request merged**